src/App.js

```diff
@@ -1,12 +1,48 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import axios from "axios";
+import PokemonCard from "./componenten/PokemonCard";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [selection, setSelection] = useState([]);
```

Comment:
Zodra de setter aangeroepen is, dan is `pokemon` niet meer van het type array. Ik zou daarom de default waarde `undefined` maken
```suggestion
    const [selection, setSelection] = useState();
```